src/interface/events.ts

```diff
@@ -23,7 +23,11 @@ export enum InterfaceEventName {
   TOKEN_SELECTED = 'Token Selected',
   TOKEN_SELECTOR_OPENED = 'Token Selector Opened',
   UNISWAP_WALLET_APP_DOWNLOAD_OPENED = 'Uniswap Wallet App Download Opened',
+  UNISWAP_WALLET_EXTENSION_DOWNLOAD_OPENED = 'Uniswap Wallet Extension Download Opened',
+  UNISWAP_WALLET_GETTING_STARTED_OPENED = 'Uniswap Wallet Getting Started Modal Opened',
+  UNISWAP_WALLET_LEARN_MORE_CLICKED = 'Uniswap Wallet Learn More Clicked',
   UNISWAP_WALLET_MICROSITE_OPENED = 'Uniswap Wallet Microsite Opened',
+  UNISWAP_WALLET_PROMO_CLOSED = 'Uniswap Wallet Promotional Modal Closed',
```

Comment:
Can we do all of these as element names / page views with trace context? We shouldn't need separate events for all of these!

Reply:
these separate events were explicitly requested by product

here's the thread for more info https://uniswapteam.slack.com/archives/C047U65H422/p1720465268995329

Reply:
would the trace context catch if these aren't separate pages but instead modals found on different pages?

Reply:
Yes they can log with the specifics on different pages! We should log all steps but we don't need unique event names for these!